### PR TITLE
ci linux: enable "Test" for Ubuntu Jammy

### PR DESCRIPTION
**Pull request by komainu8**
Because Ubuntu Jammy has already available on bento.

### CODE REVIEW

File: packages/apt/test.sh [hunk 1]
```diff
@@ -12,11 +12,20 @@ echo "debconf debconf/frontend select Noninteractive" | \
...
+  Ubuntu)
+    repository=universe
+  ;;
+esac
 code_name=$(lsb_release --codename --short)
 architecture=$(dpkg --print-architecture)
 
 wget \
-  https://packages.groonga.org/debian/groonga-apt-source-latest-${code_name}.deb
+  https://packages.groonga.org/${distribution,}/groonga-apt-source-latest-${code_name}.deb
```
Comment (kou) [inline packages/apt/test.sh:28]:
```suggestion
  https://packages.groonga.org/${distribution}/groonga-apt-source-latest-${code_name}.deb
```

File: packages/apt/test.sh [hunk 2]
```diff
@@ -66,7 +75,7 @@ Architectures: amd64 source
 SignWith: ${GPG_KEY_ID}
 DISTRIBUTIONS
 reprepro includedeb ${code_name} \
-  ${repositories_dir}/debian/pool/${code_name}/main/*/*/*_{${architecture},all}.deb
+  ${repositories_dir}/${distribution,}/pool/${code_name}/${repository}/*/*/*_{${architecture},all}.deb
```
Comment (kou) [inline packages/apt/test.sh:78]:
```suggestion
  ${repositories_dir}/${distribution}/pool/${code_name}/${repository}/*/*/*_{${architecture},all}.deb
```

File: packages/apt/test.sh (hunk 1 above)
Comment (komainu8) [reply packages/apt/test.sh:28]: `lsb_release --id --short` returns a string that is the upper case for only the first character (e.g. `Ubuntu`, `Debian`.)
Therefore, we can't use it in the URL as it is now. 
I change `$(lsb_release --id --short)` to the lower case with `{distribution,}`.

Comment (kou) [reply packages/apt/test.sh:28]: Oh, I didn't know `${XXX,}` feature!
But it's a bit tricky... How about converting `lsb_release --id --short` output?

```bash
distribution=$(lsb_release --id --short | tr 'A-Z' 'a-z')
```

**Pull request merged**